src/registrar/tests/test_admin.py

```diff
@@ -36,6 +37,57 @@
 logger = logging.getLogger(__name__)
 
 
+class TestDomainAdmin(TestCase):
+    def setUp(self):
+        self.site = AdminSite()
+        self.factory = RequestFactory()
+        self.admin = DomainAdmin(model=Domain, admin_site=self.site)
+        self.client = Client(HTTP_HOST="localhost:8080")
+        self.superuser = create_superuser()
+        self.staffuser = create_user()
+
+    def test_place_and_remove_hold(self):
```

Comment:
This will need to be expanded once the epplib stuff gets integrated. I think the best approach to do this would be to create an empty test with a skip tag on it so that work can be added later... There are examples of this in test_models_domain.py, but for your case it might look like:
```  
    @skip("Waiting on epp lib to implement")
    def test_place_and_remove_hold_epp(self):
        raise
 ```

Reply:
added this placeholder